Source/Core/Resource.js

```diff
@@ -925,6 +926,7 @@ Resource.prototype.fetchImage = function (options) {
         return Resource.createImageBitmapFromBlob(blob, {
           flipY: flipY,
           premultiplyAlpha: false,
+          skipColorSpaceConversion: true,
```

Comment:
`skipColorSpaceConversion: true` probably shouldn't be hardcoded anywhere in `Resource.js`. The option should be added to `Resource.prototype.fetchImage` and `Resource.fetchImage` and default to false. Since these functions are part of the public API the new option should be mentioned in `CHANGES.md`. The JSDoc should also be updated to reflect these new properties.

Then make sure to thread the option through the rest of the image loading functions.